packages/dynamic-links/lib/index.js

```diff
@@ -21,31 +21,25 @@ import {
   getFirebaseRoot,
 } from '@react-native-firebase/app/lib/internal';
 
-import DynamicLinkParameters from './DynamicLinkParameters';
-
+import builder from './builder';
 import version from './version';
 
-const statics = {
```

Comment:
Removing statics will mean statics is undefined on line 100, need to keep this and just empty object it.